modules/aws-kubernetes/input.tf

```diff
@@ -156,12 +158,37 @@ variable ecr-repositories {
   description = "List of docker image repositories to create"
 }
 
-data "aws_ami" "ubuntu" {
+variable s3-bucket-name {
+  default     = ""
```

Comment:
I think that we remove the default from this input variable to make it required.